### DISCUSSION TITLE
Anyone got Honeygain banned ?

### DISCUSSION THREAD

**Discussion by tm4dev** [engageub/InternetIncome#38] [General]
I ran a few containers with (my own) residential ips, but got banned after 24-48 hrs
last log:
```
2023-07-12T10:53:05.978175444Z Honeygain service disconnected
2023-07-12T10:53:23.978305684Z Honeygain service is connected and running
2023-07-12T12:13:16.638694757Z API Error: api 4xx
2023-07-12T12:13:16.978215476Z Honeygain service disconnected
2023-07-12T12:14:16.613352804Z API Error: api 4xx
2023-07-12T12:15:16.629162833Z API Error: api 4xx
2023-07-12T12:16:16.731319033Z API Error: api 4xx
2023-07-12T12:17:16.622130042Z API Error: api 4xx
2023-07-12T12:18:16.624900389Z API Error: api 4xx
```
Has anyone got the same issue? I believe this is their HAC fault

**Comment by engageub** [comment]
If you have referrals who add multiple devices they sometimes assume its your own account and ban. Please check with honeygain support and they will respond on this.
Thank you

**Comment by tm4dev** [comment]
@engageub bro, I understand your point, just asked here to know if anyone got banned like me. I don't have any referrals.
I'm waiting for their response.

**Reply by engageub** [reply]
Hi,
Did you reboot the host anytime while using it? If so, there is a chance that it might be the reason.
You can download the latest version and see. Even if you reboot the host, you won't have issue with DNS.
Thank you

**Reply by ppzzAn** [reply]
Got ban after 24-48 hours, fresh accounts with no referrals. They notice somehow with their anti fraud system.

**Comment by engageub** [comment]
You can drop an email to honeygain for the reason.
May be your ip address range is problem.

**Reply by engageub** [reply]
Hi,
Please see the discussion here. 

https://github.com/engageub/InternetIncome/discussions/71


Thank you

**Comment by af0rte** [comment]
How nany devices(Proxies) i can use  with honeygain, in my case i have 50 proxies, how i can use this number of proxies?

**Reply by af0rte** [reply]
I have put 10 proxy, in the logs i have this 
![image](https://github.com/engageub/InternetIncome/assets/36008040/005a4156-4fd0-47d8-9a65-761e3b3adc07)

but on Honeygain dashboard, i verify always ten connections are blue and doesn´t have trafic.
![image](https://github.com/engageub/InternetIncome/assets/36008040/3f7c00d6-a064-4416-a496-eab2ad82176e)

Can help me!

**Reply by engageub** [reply]
Hello,
For honeygain, you have to use only 10 proxies for single account.  If you keep getting "Waiting for traffic" even after 4 hours. 
You have to change the text honeygain/honeygain to honeygain/honeygain:0.6.6 in internetIncome.sh file wherever it is present.
Please see the discussion for more details https://github.com/engageub/InternetIncome/discussions/71 


Thank you

**Reply by af0rte** [reply]
I made the change you mentioned and it worked!
thanks again:)

**Reply by engageub** [reply]
Hello,
Just an update with respect to the recent run on honeygain.
"Connected .. waiting for traffic" may also indicate that the proxies are considered non residential IP.

Thank you
